src/bauhaus/bauhaus.c

```diff
@@ -442,10 +442,10 @@ void dt_bauhaus_init()
   gtk_widget_set_name(darktable.bauhaus->popup_area, "bauhaus-popup");
 
   darktable.bauhaus->line_space = 2;
-  darktable.bauhaus->line_height = 11;
-  darktable.bauhaus->marker_size = 0.3f;
-  darktable.bauhaus->label_font_size = 0.6f;
-  darktable.bauhaus->value_font_size = 0.6f;
+  darktable.bauhaus->line_height = 13;
```

Comment:
What do the changes in bauhaus.c have to do with the german translation?

Reply:
Something's gone wrong. Will delete the pull request and create a new one.